build.sh

```diff
@@ -401,6 +410,15 @@ build_one() {
                 INCLUDE_PATH="${root}/include" \
                 LIBRARY_PATH="${root}/lib" install
         ;;
+    libzip)
+	do_cmake \
```

Comment:
Indentation is off by one space.  (Sorry.)

What is the advantage to CMake over Autoconf?